packages/space-opera/editor/index.html

```diff
@@ -20,6 +20,8 @@
   <meta charset="utf-8">
   <meta name="description" content="&lt;model-viewer&gt; editor">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta property="og:title" content='﹤model-viewer﹥ Editor - Edit your 3D asset files for AR' />
+  <meta property="og:description" content="A tool to edit glTF and glb files ✏️, and configure them to use the ﹤model-viewer﹥ AR web component 🌐" />
```

Comment:
Let's replace `AR` with `3D/AR` or `3D & AR`, since we're actually used much more for 3D on the web, while AR is just a feature for a subset of devices.